### ISSUE TITLE
Improve responsiveness

### ISSUE THREAD

**Issue by mcdonnelldean** [nodezoo/nodezoo-web#59]
It seems there are a lot of 'inbetween' sizes where the site is two small or has too much padding as the size decreases. It looks like we need more comprehensive responsive points.

**Comment by Georgette** [comment]
Agreed, the currently layout is a generic layout template and not conditioned towards the natural breaks in the layout. I'd like to get all the info we want to display on the screen, then redo layout in general. I'm working towards that now.

**Comment by Georgette** [comment]
In addition to this, I want to redo the layout once we get most of the information we want on the interface

**Comment by Georgette** [comment]
Looking at this in more detail, we need to reduce the bloat of the html along with implementing an actual grid with breakpoints. We have a grid file, but its not really being used.

**Comment by mcdonnelldean** [comment]
@Georgette I'll assign to you since you're already doing #70. @vislamov bear in mind changes coming here since you are working on the search screen.

**Comment by vislamov** [comment]
I'm keeping it in mind.
